lib/virt_autotest/common.pm

```diff
@@ -276,6 +294,9 @@ if (get_var("REGRESSION", '') =~ /xen/) {
         if (check_var('ENABLE_SEV_SNP', '1')) {
             @allowed_guests = qw(sles15sp6efi sles15sp7efi);
         }
+        if (check_var('ENABLE_SEV_ES', '1')) {
```

Comment:
15SP6 product also needs sev-es test, and it is quite different from 15sp7 from the regards of modules containing the pkg as well as pkg names. It's given in the ticket description.